marathon_lb.py

```diff
@@ -226,7 +227,8 @@ def tasks(self):
         return self.api_req('GET', ['tasks'])["tasks"]
 
     def get_event_stream(self):
-        url = self.host + "/v2/events"
+        self.current_host = self.host
+        url = self.current_host + "/v2/events"
```

Comment:
I'd rather we still use self.host here as there is another pending change that also touches this line and depending on ordering there will be conflicts.